API/src/microservices/Database/bookshelf.js

```diff
@@ -0,0 +1,13 @@
+//const { database } = require("../../../config.example.json");
+
+const knex = require("knex")({
+  client: "pg",
+  connection: {
+    host: "localhost",
+    user: "postgres",
+    password: "password",
+    database: "risk",
+  },
+});
```

Comment:
```suggestion
const knex = require("knex")({
  client: "pg",
  connection: {
    host: "localhost",
    user: "postgres",
    password: "password",
    database: "risk",
  },
});
```

Yeah this is a really bad idea.  It's fine because this is just an example project, but if these were real-world database credentials, it would be a big security risk.

Reply:
That's why we keep the config locally and import from it.  The server would have it's own local copy of the config.json

Reply:
I understand. Will go back and change it. Do we need to hash the password as well?

Reply:
Nope, this password can be whatever you want.